dimensions: # for yellow porter ref https://trucks.cardekho.com/en/trucks/piaggio/porter-700/specifications
  overall_length    : 3.544 # all the mesurments in MKS(meters, kgs, seconds)
  overall_width     : 1.460
  overall_height     : 1.75
  wheel_base        : 1.82
  track_width       : 1.40
  front_overhang    : 1.0
  rear_overhang     : 0.7
  ground_clearance  : 0.2
  payload           : 750 # kgs
  tyre_radius       : 0.3
  tyre_section_width :  0.145
  gear_ratio         : 26.67

motion_limits:
  max_forward_speed      : 3  # m/s
  min_forward_speed      : 0.3 # m/s
  max_backward_speed : -3
  min_backward_speed : -0.3
  max_acceleration : 0.5
  min_acceleration : -0.5
  max_steering_angle : 30 # degrees
  min_steering_angle : -30 # degrees

robot_origin_frame: "base_link" # name of link at rear wheel

base_link:   # center of the rear axle.
  gps_link:  # center of the rear axle to gps position, center point of two gps modules.
    x: 2.080
    y: 0.0
    z: 0.0
    roll: 0.0
    pitch: 0.0
    yaw: 0.0
  zed_camera: # center of the rear axle to zed camera 
    x: 2.8
    y: 0.0
    z: 0.5
    roll: 0.0
    pitch: 0.0
    yaw: 0.0

  rslidar: # center of the rear axle to rslidar(3d lidar)
    x: 2.8
    y: 0.0
    z: 0.2
    roll: 0.0
    pitch: 0.0
    yaw: 0.0

  rplidar: # center of the rear axle to rplidar (2d lidar )
    x: 2.8
    y: 0.0
    z: 0.1
    roll: 0.0
    pitch: 0.0
    yaw: 0.0